src/Pickles/Pickles.DocumentationBuilders.Markdown/Stylist.cs

```diff
@@ -0,0 +1,168 @@
+﻿//  --------------------------------------------------------------------------------------------------------------------
+//  <copyright file="Stylist.cs" company="PicklesDoc">
+//  Copyright 2018 Darren Comeau
+//  Copyright 2018-present PicklesDoc team and community contributors
+//
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//  </copyright>
+//  --------------------------------------------------------------------------------------------------------------------
+
+using PicklesDoc.Pickles.ObjectModel;
+
+namespace PicklesDoc.Pickles.DocumentationBuilders.Markdown
+{
+    class Stylist
+    {
+        public string TableResultHeading { get { return "Result"; } }
+
+        internal virtual string AsTitle(string title)
+        {
+            var titleTemplate = "# {0}";
+
+            var styledTitle = string.Format(titleTemplate, title);
+
+            return styledTitle;
+        }
+
+        internal virtual string AsFeatureHeading(string featureName)
+        {
+            var featureHeadingTemplate = "### {0}";
```

Comment:
I think it would be nice if it followed markdownlint rule [MD001 heading-increment/header-increment - Heading levels should only increment by one level at a time.](https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md001---header-levels-should-only-increment-by-one-level-at-a-time)